packages/admin/admin/src/dataGrid/excelExport/DataGridExcelExportButton.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, CircularProgress } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { FileIcon } from "../../fileIcons/FileIcon";
+import { ExportApi } from "./useDataGridExcelExport";
+
+interface Props {
+    exportApi: ExportApi;
+    onClick?: () => void;
+    loadingComponent?: React.ReactNode;
+    onError?: () => void;
+}
+
+export const DataGridExcelExportButton: React.FunctionComponent<Props> = ({ onClick, children, exportApi, loadingComponent, onError }) => {
+    const onClickButtonPressed = () => {
+        if (onClick != null) {
+            onClick();
+        } else {
+            exportApi.exportGrid();
+        }
+    };
+
+    const { loading, error } = exportApi;
+
+    if (error && onError) {
+        onError();
+    }
+
+    return (
+        <Button
```

Comment:
It bugs me that the text of this button is blue. Please make it black. You can do that simply by adding `color="info"`